environment.yml

```
@@ -4,24 +4,25 @@ channels:
  - anaconda
  - defaults
 dependencies:
+- gdal
+- rasterio
+- affine
+- Fiona
+- geopandas
+- matplotlib
+- numpy
+- pandas
+- rasterio
+- Shapely
```

```suggestion
- shapely
```
Conda package name is lowercase, but it looks like it works either way